data/Makefile.am

```diff
@@ -1,3 +1,7 @@
 iconoverridesdir = $(datadir)/eos-application-tools/icon-overrides
 
 dist_iconoverrides_DATA = eos-icon-overrides.ini
+
+flatpaksdir = $(datadir)/eos-application-tools/flatpaks
```

Comment:
It would be good to have a brief comment above this section explaining that it gives the list of flatpaks to auto-install.

Reply:
Also, `$(datadir)` here doesn’t match `$(libdir)` in the commit message. Which one is intended?